src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -757,6 +757,7 @@ export default class Pilot {
      * @return {array} [success of operation, readback]
      */
     initiateHoldingPattern(fixName, holdParameters) {
+        const radial = heading_to_string(holdParameters.inboundHeading + Math.PI);
         const cardinalDirectionFromFix = getRadioCardinalDirectionNameForHeading(holdParameters.inboundHeading);
```

Comment:
I assume this rounds the value to the nearest integer? Don't want a response of `holding east of ABBOT on the 4.174926391036391846291739 radial, right turns, 10 mile legs` :laughing:

Reply:
_"Readback correct, but plus or minus a degree will suffice"_

But yes, it does